### ISSUE TITLE
Add a Minting View

### ISSUE THREAD

**Issue by tweakch** [CardanoSharp/cardanosharp-catalyst-demo#5]
We could add a minting view where the user can mint an NFT-Selfie

**Comment by nothingalike** [comment]
yeah i definitely like the idea of having this app be a little sandbox demo of what the library can do
